### ISSUE TITLE
use "other" as a default formatRelative token

### ISSUE THREAD

**Issue by terrysahaidak** [date-fns/date-fns#1273]
I've just spent 1 hour figuring out why `date-fns` keeps me throwing an error

> RangeError: Format string contains an unescaped latin alphabet character `n`

My locale's formatRelative config is the following:
```
{
  today: "'Today'",
  other: 'MM/dd/yyyy n',
}
```

The error was here: https://github.com/date-fns/date-fns/blob/master/src/formatRelative/index.js#L94

Since I've chosen the date which returns token such as `tomorrow`, there is no `tomorrow` field in my config, so it returned `undefined` and passed it to the `locale.formatRelative` which threw an error.

My suggestion is to add some sort of default token or showing more useful error for that case. Also, would be cool to mention that in the docs.

**Comment by kossnocorp** [comment]
Thank you for reporting it. We shall throw an exception when a field is missing in the locale. Using `other` as the default will be even more confusing.

**Comment by terrysahaidak** [comment]
@kossnocorp maybe would be cool to have some sort of defaults like moment does.

**Comment by kossnocorp** [comment]
My only concern that it would make locale development confusing. A simple typo might make someone spend an hour or even more debugging why `tomorow` doesn't work.

We'll consider using defaults, anyways, thanks.

**Comment by terrysahaidak** [comment]
Makes sense for me. The part about requiring all the fields should be mentioned in docs as well :)

**Comment by kossnocorp** [comment]
I did reconsider the solution and agree that using the defaults is better because usually new locales are copied from English or other canonical locales. Thank you for your input! ❤️ 

I'm moving this to v2+ milestone as it wouldn't a breaking change. If you want to help, PR is very welcome 🙏

Thanks!

**Comment by pin4eva** [comment]
what is the solution to this, because i am facing the same problem

**Comment by scrozier** [comment]
I too have the same problem and don't understand from this thread how the locale is to be set or referenced.

**Comment by JacquesBonet** [comment]
Me too same problem.

**Comment by Gelio** [comment]
I took a stab at fixing this issue in https://github.com/date-fns/date-fns/pull/2217 🙂
